Libraries/JavaScriptAppEngine/Initialization/symbolicateStackTrace.js

```diff
@@ -11,13 +11,20 @@
  */
 'use strict';
 
-const {fetch} = require('fetch');
 const getDevServer = require('getDevServer');
 const {SourceCode} = require('NativeModules');
+// Do not require fetch immediately. See symbolicateStackTrace()
+// Workaround for https://github.com/facebook/react-native/issues/9605
```

Comment:
That's probably because it is not trivial to understand. I'll have to think about how to summarize the exact issue in just a few lines, if that is at all possible.